snowplow_tracker/tracker.py

```diff
@@ -734,6 +740,67 @@ def track_screen_view(
             event_json, context, tstamp, event_subject
         )
 
+    def track_mobile_screen_view(
+        self,
+        id_: str,
```

Comment:
Nit: it would be nice if the tracker automatically generated the UUID if the user didn't pass the id (it could be optional).